application/Apollon.Mud/Apollon.Mud.Shared/Dungeon/DungeonDto.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Apollon.Mud.Shared.Dungeon.Room;
+using Apollon.Mud.Shared.Dungeon.User;
+using Newtonsoft.Json;
+
+namespace Apollon.Mud.Shared.Dungeon
+{
+    /// <summary>
+    /// ToDo
+    /// </summary>
+    public class DungeonDto
+    {
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("DungeonName")]
+        public string DungeonName { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("DungeonDescription")]
+        public string DungeonDescription { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("DungeonEpoch")]
+        public string DungeonEpoch { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("DefaultRoom")]
+        public RoomDto DefaultRoom { get; set; }
+
+        /// <summary>
+        /// ToDo 
+        /// </summary>
+        [JsonProperty("Visibility")]
+        public int Visibility { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("Id")]
+        public Guid Id { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("Status")]
+        public int Status { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("DungeonMasters")]
+        public ICollection<DungeonUserDto> DungeonMasters { get; set; }
+
+        /// <summary>
+        /// ToDo
+        /// </summary>
+        [JsonProperty("CurrentDungeonMaster")]
+        public DungeonUserDto CurrentDungeonMaster { get; set; }
```

Comment:
Spielt das bei der Konfiguration eine Rolle?

Reply:
Konfiguration an sich nein, wäre aber die frage, ob man des nicht in der Ansicht der Dungeon-Auswahl nicht anzeigen lassen will, wer aktuell DM ist. Kann sonst weggelassen werden.